entities/library.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative '../entities/book'
+require_relative '../entities/reader'
+require_relative '../entities/author'
+require_relative '../entities/order.rb'
+require_relative '../tasks/seed'
+
+class Library
+  attr_reader :readers, :orders, :authors, :books
+
+  include Statistics
+
+  def initialize(readers:, orders:, authors:, books:)
+    @readers = readers
+    @orders = orders
+    @authors = authors
+    @books = books
+  end
+end
```

Comment:
please add a new line
rubocop extension in your code editor should highlight such issues

Reply:
done